src/crystal/init_project/template/gitignore.ecr

```diff
@@ -0,0 +1,4 @@
+/.deps/
+/.deps.lock
```

Comment:
Maybe we can have `crystal init lib example` and `crystal init app example`. If you write `crystal init` it will tell you to choose `app` or `lib`.